test/addons/openssl-client-cert-engine/test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+require('../../common');
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+const path = require('path');
+
+var engine = path.dirname(module.filename) + '/build/Release/testengine.so';
```

Comment:
Also, it is very likely that `.so` is going to be something different on non-linux platforms.